content/en/quickstart/create-device.md

```diff
@@ -11,3 +11,105 @@ menu:
     weight: 10
 toc: true
 ---
+This guide will cover how to get the first steps while using the plgd cloud in the Raspberry Pi. This includes how to create cloud server (OCF Device) available from IoTivity-Lite open source project on the Github.
+
+## Requirements
+You will need the following: 
```

Comment:
If it's section requirements, you don't need to explain that "you need the following". You are just repeating the header.